web/src/pages/project/ProjectSummary.tsx

```diff
@@ -81,6 +84,44 @@ const ProjectSummary: React.FunctionComponent = () => {
         [projectName, navigate]
     )
 
+    const updateFilters = React.useCallback(
+        (e) => {
+            if (!summary) return
+            /* eslint-disable no-param-reassign */
+            const processedFilter = Object.entries(e).reduce((filter, [column, v]) => {
+                if (!v) {
+                    return filter
+                }
+                const query = v as string
+                const participantIndex = summary.participant_keys.findIndex((i) => i[1] === column)
```

Comment:
If you're filtering on a meta field that also is a first class field - this finds the non-meta field, but also causes both to highlight:

<img width="685" alt="image" src="https://user-images.githubusercontent.com/22381693/229020065-09a5e370-2dd9-4995-9e4f-4c87dd4d0f46.png">